mautic/mautic
# add safari support for state drop down

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Adds safari support for state dropdown



#### Description:

Forms with country and state fields do not correctly filter out states when selecting a country in safari

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a form with country and state field
    - The State select field needs to be added via the Add field based on selected value.
    - The condition should be any.
    - Mapped should be set to the State.
    - Use assigned contact/company field's list choices should be toggled on.
4. in safari select a country and verify the state dropdown only shows states for that country.
